test/database.rb

```diff
@@ -1,4 +1,12 @@
-connection_options = { adapter: "mysql2", host: "127.0.0.1", username: "root" }
+require 'uri'
+mysql_url = URI(ENV['MYSQL_URL'].to_s)
```

Comment:
simpler and shows the expected format ?
```suggestion
mysql_url = URI(ENV['MYSQL_URL'] || "root@127.0.0.1")
```

Reply:
🤦🏼 of course. Thank you.